application/classes/Ushahidi/Repository/Post.php

```diff
@@ -689,9 +701,13 @@ public function getGroupedTotals(SearchData $search)
 
 		// Fetch the results and...
 		$results = $this->search_query->execute($this->db);
+		$results = $results->as_array();
+
+		// Append unmapped totals to
+		$results['unmapped'] = $this->getUnmappedTotal($this->getSearchTotal());
```

Comment:
This is kind of weird adding this to all the stats responses (ie. regardless of what we're grouping by... but I don't have a better solution particularly.

Does this mess with the activity view at all? (hopefully our code was robust and it doesn't)